src/services/dashboards/dashboards.svc.coffee

```diff
@@ -156,11 +155,18 @@ angular
       )
 
 
+    # Will be filled only once
```

Comment:
this comment was to explain line `return false if _.isEmpty(templatesArray) || !_.isEmpty(_self.config.widgetsTemplates)`, not the all method